bayeux.go

```diff
@@ -266,7 +295,12 @@ func (b *Bayeux) connect(ctx context.Context, out chan MaybeMsg) chan MaybeMsg {
 						return
 					}
 					for _, e := range x {
-						out <- MaybeMsg{Msg: e}
```

Comment:
Could you please add any comments regarding this change?

Reply:
Sure, Thanks